layouts/docs/list.html

```diff
@@ -24,7 +24,11 @@
             <h1>{{ .Title }}</h1>
             {{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
         {{ end }}
-        {{ partial "section-index.html" . }}
+        {{- if eq .Page.RelPermalink "/docs/reference/kubectl/generated/" -}}
```

Comment:
We shouldn't hard code this path. Instead, set a custom layout in the front matter of that page.

Reply:
Acknowledged! The setting of custom layout in the front matter of that page would need to done in the generator code _(as this is an auto-generated index page)_.